valdr-ng/projects/valdr-ng-showcase/src/app/simple-form/simple-form.component.ts

```diff
@@ -15,6 +15,7 @@ export class SimpleFormComponent implements OnInit {
   personForm!: FormGroup;
 
   constructor(private fb: FormBuilder) {
+    this.person = {} as any;
```

Comment:
it's anyway a showcase, maybe you can add some dummy values instead of casting as any here

Reply:
`this.person` is an input, and this is to circumvent the linting issues of possibly undefined.